packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -112,12 +116,13 @@ export class QuoteRequestor {
                         timeout: _opts.makerEndpointMaxResponseTimeMs,
                     });
                 } catch (err) {
-                    logUtils.warn(
+                    logUtils.log(`failing`);
```

Comment:
I assume this was a debug line that could be removed?

Reply:
:scream: thank you very much, removed in https://github.com/0xProject/0x-monorepo/pull/2562/commits/141c30b173881f1f09307d951d6d8a63483f732e